GUI2.py

```diff
@@ -754,6 +762,41 @@ def get_monster():
         # opp_att = random.randint(50, 60)
 
 
+def get_bossmonster():
+    global m
+    global opp_hp
+    global monster
+    global frame_monster_1
+    frame_monster_1 = Frame(root)
+    frame_monster_1.pack()
+    L_monster_Wel = Label(frame_monster_1, text="You have to fight a boss monster.")
+    L_monster_Wel.pack()
+    bossmonsters = ("Demon Slayer","Big Tooth")
+    monster = random.choice(bossmonsters)
+    # print(monster)
+    opp_hp = 100
+    if monster == "Demon Slayer":
+        m = 1
+        # monster 1
+        # Attack in range of 0-10
+        L_m1_intro = Label(frame_monster_1, text="You have to face Goblin\n"
+                                                 "The match starts. You get the first chance\n")
+        L_m1_intro.pack()
+        fight_monster()
+        # opp_att = random.randint(0, 10)
+
+    if monster == "Big Tooth":
+        m = 2
+        # monster 2
+        # Attack in range of 10-20
+        L_m2_intro = Label(frame_monster_1, text="You have to face Werewolf\n"
+                                                 "The match starts. You get the first chance\n")
+        L_m2_intro.pack()
+        fight_monster()
+        # opp_att = random.randint(10, 20)
+
+
+
```

Comment:
Call the quit function after get_bossmonster is executed or make a custom ending screen(preferable)